autobahn/autobahn/asyncio/wamp.py

```diff
@@ -98,13 +100,19 @@ def _reject_future(future, value):
         future.set_exception(value)
 
     @staticmethod
-    def _add_future_callbacks(future, callback, errback):
+    def _add_future_callbacks(future, callback, errback,
```

Comment:
I can just use currying `add_future_callbacks(future, lambda x: callback("foo", x))` which is more flexible, since it allows me to chose the order between actual result and additional parameters to callback.